readme.md

```diff
@@ -118,47 +122,49 @@ Once installation is working, try creating and requesting resources. Note that t
 
 * **Question:** The response you receive is _not_ the document you updated. What information is being represented here? Try replacing `Model.updateOne()` with the [Model.findOneAndUpdate()](https://mongoosejs.com/docs/api.html#model_Model.findOneAndUpdate) and see the difference in the result.
 
-* **Your Answer:**
+* **Your Answer:** It seems like `Model.updateOne()` will return back meta data about the request. `Model.findOneAndUpdate()` will return back the old version of the document that was updated.
 
 * **Question:** This new method will return the _old document_ instead of the new one. What option can you add to your statement to return the new one?
 
-* **Your Answer:**
+* **Your Answer:** We can pass in the `new` option, which if true, will return the modified document.
 
 * **Question:** Another field was updated when you ran this command. Which one was it?
 
-* **Your Answer:**
+* **Your Answer:** `updated_at` was updated with a new time stamp.
 
 * **Question:** Take a look at the terminal window running your server. You are likely getting the following deprecation warning:
+
   ```
   DeprecationWarning: Mongoose: `findOneAndUpdate()` and `findOneAndDelete()` without the `useFindAndModify` option set to false are deprecated. See: https://mongoosejs.com/docs/deprecations.html#-findandmodify-
   ```
+
   Take a look [at this page](https://mongoosejs.com/docs/deprecations#findandmodify) to see how to fix it. Describe the changes that took place.
 
-* **Your Answer:**
+* **Your Answer:** Adding `useFindAndModify: false` to `mongoose.connect()` in our `app.js` file makes the error in the terminal disappear.
 
 ---
 
 - [ ] Update the API so that you can successfully delete a record. Use the [Model.findOneAndDelete()](https://mongoosejs.com/docs/api.html#model_Model.findOneAndDelete) method.
 
 ---
 
-- [ ] A lot of information is being returned on each request. Some of this information we may want to keep private. Using the [.select()](https://mongoosejs.com/docs/api.html#query_Query-select) method, return only the "_id", "title", "start_year", and "season_count" on all the above requests.
+- [ ] A lot of information is being returned on each request. Some of this information we may want to keep private. Using the [.select()](https://mongoosejs.com/docs/api.html#query_Query-select) method, return only the "\_id", "title", "start_year", and "season_count" on all the above requests.
 
 * **Question:** At least one of these will throw an error. Which one? How can you get around this issue?
 
-* **Your Answer:** 
+* **Your Answer:** `POST` threw a 400 error because the server could not process the request being sent by the client. I've tried a number of the different query methods in the Mongoose docs to see if I could get around this and wasn't able to. Hoping we can cover the solution with more detail in class.
```

Comment:
Right, you should encounter an error when you run `.create().select()`. You will have to re-request the information or use the `.save()` method to return the updated record. Let me know if you're still having trouble doing this!